content/applications/websites/livechat/ratings.rst

```diff
@@ -2,44 +2,138 @@
+icon, a text box will appear. Customers can add comments in this text box to explain why they chose
+this rating. This message will be sent to the live chat operator, along with the rating icon.
+
+.. image:: ratings/live-chat-ratings-operator-window.png
+   :align: center
+   :alt: View of a chat window from an operator's window highlighting a rating for Odoo Live Chat.
+
+Publish customer ratings
+========================
+
+.. note::
+   The customer notes that are submitted with the rating will *not* be published on the website.
+   These are kept internal. Only a statistical overview of the performance of operators for the
+   *channel*.
+
+To publish a channel's ratings on the website, first navigate to a live chat channel's record by
+going to the :menuselection:`Live Chat` app and clicking on the kanban card for that team. Then
+click on the :guilabel:`Go to Website` smart button. This will open the :guilabel:`Live Chat Channel
+Statistics` page.
+
+In the upper right corner of the page, click the red :guilabel:`Unpublished` slider to read
+:guilabel:`Published`.
+
+.. image:: ratings/live-chat-ratings-unpublished.png
    :align: center
-   :height: 400
-   :alt: View of the chat window from a user’s side for Odoo Live Chat
+   :alt: View of the published ratings on the portal for Odoo Live Chat.
 
-The rating is shown on the chat window itself for the operator.
+Add ratings page to site
+------------------------
 
-.. image:: ratings/operator_chatwindow.png
+Once the rating page has been published, it has to be added to the website. To do this, go to
+:menuselection:`Dashboard --> Website --> Site --> Pages --> New`.
+
+This will open a :guilabel:`New Page` window. In the :guilabel:`Page Title` field, enter `livechat`.
+Click :guilabel:`Create`.
+
+.. important::
+   The page *must* be named `livechat` in order for the database to recognize and connect the
+   ratings page. The page title can be changed later under the :guilabel:`Menu Editor`.
+
+The newly created webpage will open, with the :guilabel:`Webpage Editor` open in the right panel.
+The page will list out the names of the :guilabel:`Live Chat Channels` whose ratings pages have been
+published. Next to the channel name is a speech bubble icon, which users can click on to be taken to
+the ratings' page for the respective channel.
+
+.. image:: ratings/live-chat-published-icon.png
+   :align: center
+   :alt: View of the webpage for Live Chat ratings emphasizing the channel icon.
+
+Make any desired changes or additions to this page, then click save on the :guilabel:`Webpage
+Editor`.
+
+To publish the `livechat` webpage, return to the list of webpages by navigating to
+:menuselection:`Site --> Pages`. Click the checkbox to the left of `livechat` in the list of pages
+to select the page. Then, click the checkbox under the column labeled :guilabel:`Is Published`. Then
+click :guilabel:`Save`.
```

Comment:
When I do this, the flow is different. After clicking the checkbox next to livechat, it highlights the line. Then, I click on the box next to Is Published- the first click just 'highlights' that box (it turns white) but it does not check the box. The second time I click, the box is checked off, but then there is no Save option- you only see 'new' where 'save' previously was, because doing the double check makes the page published. If I tried to click once, and it is not checked off and click save nothing happens.